src/components/AddMovie/AddMovieComponent.js

```diff
@@ -65,145 +38,152 @@ export default class AddMovieComponent extends Component {
 
   render() {
     const { editMode, onSubmit } = this.props;
-    const {
-      movie,
-      movie: { id, title, releaseDate, movieURL, genre, overview, runtime }
-    } = this.state;
-    const errors = this.isDataValid();
-    const isSubmitDisabled = Object.keys(errors).length > 0;
+    const movie = this.props.movie || DEFAULT_MOVIE;
 
     return (
-      <>
-        <div className="addmovie-header">
-          {
-            editMode
+      <Formik
+        initialValues={movie}
+        validate={values => {
+          return this.isDataValid(values);
+        }}
+        onSubmit={(values, { setSubmitting }) => {
+          values.runtime = Number(values.runtime);
```

Comment:
why did you convert number value with Number constructor, if this value not a number pls do it averywhere!

Reply:
even input type number , when we read from input it is giving string (4 as '4') but api strictly accepting numbers